ramenya2.py

```diff
@@ -389,13 +390,21 @@ def submit(args):
         elif len(args.runs)==3: kaedama = kaedama.bake( "--runs", args.runs[0], args.runs[1], args.runs[2] )
         else:                   kaedama = kaedama.bake( "--runs", "0", "999999" )
 
-        # Execute the specified rules
-        for r in args.rules:
-            kaedama( batch=True, rule=r, _out=sys.stdout )
+        list_of_active_rules = args.rules
```

Comment:
On each iteration of the submission loop, read list of rules from a file.  Enables dynamic changes to the productions being managed.